bigramProbability.py

```diff
@@ -0,0 +1,44 @@
+import re
+import csv
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+from nltk.probability import FreqDist
+from nltk import ngrams
+from nltk.corpus import movie_reviews
+from collections import Counter
+
+bigramsDictionary = Counter()
+unigramsDictionary = Counter()
+bigramsProbability = {}
+
+all_words = []
+for w in movie_reviews.words():
+	all_words.append(w.lower())
+
+new_words = []
+for w in all_words:
+	if w.isalpha():
+		new_words.append(w)
+
+stopWords = set(stopwords.words("english"))
+character = {'u'}
+tokensFiltered = []
```

Comment:
```suggestion
tokens_filtered = []
```

Again, some PEP-8 comments. Even ignoring that, our focus is on consistency. So since you have already defined other variables such as `all_words` or `new_words`, you should stick to that convention.

And, of course, we reserve camelCase to methods

Reply:
Totally agree. I had a problem with the conventions back then. Now I think I am fine.